<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameScreen</class>
 <widget class="QWidget" name="GameScreen">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>536</width>
     <height>419</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="topMargin">
       <number>30</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <spacer name="verticalSpacer">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="PointsBar" name="healthLvl">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignTop">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Health</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <spacer name="verticalSpacer_6">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="PointsBar" name="satietyLvl">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignTop">
         <widget class="QLabel" name="label_2">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Satiety</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <spacer name="verticalSpacer_7">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="PointsBar" name="happinessLvl">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignTop">
         <widget class="QLabel" name="label_3">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Happiness</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <spacer name="verticalSpacer_8">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="PointsBar" name="energyLvl">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignTop">
         <widget class="QLabel" name="label_4">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Energy</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_5">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_9">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="toFeed">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>17</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Feed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="toSleep">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>17</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Sleep</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="toWakeUp">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>17</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Wake up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="toWash">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>17</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Wash</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="toCure">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>17</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Cure</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="GameWidget" name="widget1" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Age:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="ageLbl">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Birth:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="birthLbl">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="toMainBtn">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>17</pointsize>
         </font>
        </property>
        <property name="text">
         <string>To main window</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GameWidget</class>
   <extends>QWidget</extends>
   <header location="global">gamewidget.h</header>
   <container>1</container>
   <slots>
    <signal>chHealthSgn(int)</signal>
    <signal>chSatSgn(int)</signal>
    <signal>chHapSgn(int)</signal>
    <signal>chEnSgn(int)</signal>
    <signal>nowSleep()</signal>
    <slot>toFeed()</slot>
    <slot>toSleep()</slot>
    <slot>toWash()</slot>
    <slot>toCure()</slot>
    <slot>toWakeUp()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>PointsBar</class>
   <extends>QPushButton</extends>
   <header location="global">pointsbar.h</header>
   <slots>
    <slot>setLevel(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>widget1</sender>
   <signal>chEnSgn(int)</signal>
   <receiver>energyLvl</receiver>
   <slot>setLevel(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>521</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widget1</sender>
   <signal>chHapSgn(int)</signal>
   <receiver>happinessLvl</receiver>
   <slot>setLevel(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widget1</sender>
   <signal>chHealthSgn(int)</signal>
   <receiver>healthLvl</receiver>
   <slot>setLevel(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>59</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widget1</sender>
   <signal>chSatSgn(int)</signal>
   <receiver>satietyLvl</receiver>
   <slot>setLevel(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toWash</sender>
   <signal>clicked()</signal>
   <receiver>widget1</receiver>
   <slot>toWash()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toWakeUp</sender>
   <signal>clicked()</signal>
   <receiver>widget1</receiver>
   <slot>toWakeUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toSleep</sender>
   <signal>clicked()</signal>
   <receiver>widget1</receiver>
   <slot>toSleep()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toFeed</sender>
   <signal>clicked()</signal>
   <receiver>widget1</receiver>
   <slot>toFeed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toCure</sender>
   <signal>clicked()</signal>
   <receiver>widget1</receiver>
   <slot>toCure()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
